### PR TITLE
fix(ui): fixed ui issues

### PR DESCRIPTION
**Pull request by yaniryos**
- fixed bug 366
- fixed bug 318
- fixed bug 359

### CODE REVIEW

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts [hunk 1]
```diff
@@ -4,23 +4,11 @@ const useStyles = makeStyles({
...
         flexDirection: 'row',
-        justifyContent: 'space-between',
         marginLeft: '-5.5vw',
         '@media screen and (max-width: 1280px)': {
             marginLeft: '-20.5vw'
         },
     },
-    airportCountryTextField: {
-        width: '10vw',
-        '@media screen and (min-width: 1450px)': {
```
Comment (daniel7764) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts:15]: what about responsiveness if you deleted the media query?

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts [hunk 2]
```diff
@@ -4,23 +4,11 @@ const useStyles = makeStyles({
...
-            marginRight: '-6vw',
-        },
-    },
-    additionalAirportDetails: {
-        '@media screen and (min-width: 1800px)': {
```
Comment (daniel7764) [inline client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts:20]: same here, i didn't put the divs you deleted for nothing

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts (hunk 1 above)
Comment (yaniryos) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts:15]: it wasnt used

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts (hunk 2 above)
Comment (yaniryos) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts:20]: it wasnt used

File: client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts (hunk 1 above)
Comment (yaniryos) [reply client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AirportInputStyles.ts:15]: my bad will fix it

**Review by daniel7764 — APPROVED** [review]

**Review by roymaniv — APPROVED** [review]

**Pull request merged**